### ISSUE TITLE
Homepage setting in package.json not working as expected

### ISSUE THREAD

**Issue by tatsuhirosatou** [egoist/vbuild#135]
**Do you want to request a *feature* or report a *bug*?**

Report a bug

**What is the current behavior?**

Paths on /dist/index.html do not include the correct path for resources

**If the current behavior is a bug, please provide the steps to reproduce.**

Set "homepage": "/kreader/static/", in package.json

Run: vbuild index.js

**What is the expected behavior?**

/kreader/static would be appended to resource paths at index.html

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, vbuild version and Operating System.**

If I set  homepage: '/kreader/static/' at vbuild.config.js then it works as expected

vbuild --version
6.20.1

**Comment by egoist** [comment]
Can you provide a minimal repo to reproduce this? It's working for me.

**Comment by tatsuhirosatou** [comment]
I got it working. The problem was I was running vbuild index.js on the src directory. When I ran yarn build on the root directory it worked. Perhaps this should be on the documentation?

Thanks a lot for this project. This is great. I hate having to deal with webpack configs and such, I don't know how to use them I am not interested in learning. I just want to build my Vue project with sane defaults, without having to maintain endless config files. This is exactly what I needed.
